kolibri/plugins/learn/assets/src/views/TopicsPage/index.vue

```diff
@@ -542,9 +541,11 @@
           return 4;
         } else return null;
       },
+      throttledStickyCalculation() {
+        return throttle(this.stickyCalculation);
+      },
       // calls handleScroll no more than every 17ms
-      throttledHandleScroll() {
-        throttle(this.stickyCalculation);
+      throttledtabPositionCalculation() {
```

Comment:
Non blocking, but would be good to keep this as fully camel case for better readability. It would turn to `throttledTabPositionCalculation`

Reply:
```suggestion
      throttledTabPositionCalculation() {
```